app/screens/settings/display_settings/display_settings.js

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {intlShape} from 'react-intl';
+import {
+    Platform,
+    View
+} from 'react-native';
+
+import SettingsItem from 'app/screens/settings/settings_item';
+import StatusBar from 'app/components/status_bar';
+import {wrapWithPreventDoubleTap} from 'app/utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import ClockDisplay from 'app/screens/clock_display';
+
+export default class DisplaySettings extends PureComponent {
+    static propTypes = {
+        navigator: PropTypes.object.isRequired,
+        intl: intlShape.isRequired,
+        theme: PropTypes.object.isRequired
+    };
+
+    static contextTypes = {
+        intl: intlShape
+    };
+
+    state = {
+        showClockDisplaySettings: false
+    };
+
+    goToClockDisplaySettings = wrapWithPreventDoubleTap(() => {
+        const {navigator, theme} = this.props;
+        const {intl} = this.context;
+
+        if (Platform.OS === 'ios') {
+            navigator.push({
+                screen: 'ClockDisplay',
+                title: intl.formatMessage({id: 'mobile.advanced_settings.clockDisplay', defaultMessage: 'Clock Display'}),
```

Comment:
If i'm correct there should be already a string for `Clock Display` defined as `user.settings.display.clockDisplay`